pkg/core/construct_graph_test.go

```diff
@@ -317,7 +317,7 @@ func Test_LoadConstructsIntoGraph(t *testing.T) {
 					{Provider: AbstractConstructProvider, Type: "orm", Name: "test_orm"},
 					{Provider: AbstractConstructProvider, Type: "kv", Name: "test_kv"},
 					{Provider: AbstractConstructProvider, Type: "fs", Name: "test_fs"},
-					{Provider: AbstractConstructProvider, Type: "secret", Name: "test_secret"},
+					{Provider: AbstractConstructProvider, Type: "config", Name: "test_secret"},
```

Comment:
There's non-config secrets (for Python and JS) right?

Reply:
yeah, you can still pass in secrets, but we dont have a way to pass in the actual secrets paths yet since we only pass in id, which is why i use config.

Its something well need to likely support outside of infracopilot but we need to discuss how we want to do that for importing when we are importing more than just the id and the parsing gets much trickier

Reply:
i guess that could be the attributes part of a constructConstraint?